app/routes/joblistings/components/JoblistingList.css

```diff
@@ -51,9 +51,11 @@
 .companyLogo {
   width: 120px;
   height: 80px;
-  background: #eee;
+
+  /* background: #eee; */
```

Comment:
delete the comment if you don't need it?

Reply:
:+1: